### ISSUE TITLE
시스템18-1 성능실험

### ISSUE THREAD

**Issue by bart2001** [nlp-research/biaffineparser#64]
시스템 특징
- Pre-trained 단어/태그 embedding (200-dim, trainble=False, embeddings/words.pos.original.vec)
- Random 음절 embedding (200-dim, trainble=True)
- Random 태그 embedding (100-dim, trainble=True)
- Bi-LSTM at embedding_layer
- num_lstm_layer=4 (시스템18과 다른부분, https://github.com/nlp-research/biaffineparser/issues/60)

* 학습데이터에 ;이 포함된 문장에 대해서 코멘트 처리가 되어서 nan으로 해석된 사례가 일부 있음 (sejong.char.train.csv 내에 한 어절 기준 26건)

**Comment by bart2001** [comment]
hparams
```python
{
    "num_lstm_units": 400,
    "num_lstm_layers": 4,
    "arc_mlp_units": 500,
    "label_mlp_units": 100,
    "word_embed_size": 200,
    "char_embed_size": 200,
    "pos_embed_size": 100,
    "optimizer": "adam",
    "learning_rate": 0.001,
    "decay_factor": 0.9,
    "num_train_epochs": 100,
    "train_filename": "data/sejong.char.train.csv",
    "dev_filename": "data/sejong.char.test.csv",
    "out_dir": "parser/model_18_1",
    "word_vocab_name": "word.pkl",
    "char_vocab_name": "char.pkl",
    "pos_vocab_name": "pos.pkl",
    "rel_vocab_name": "rel.pkl",
    "head_vocab_name": "head.pkl",
    "word_embed_file": "embeddings/words.pos.original.vec",
    "char_embed_file": "",
    "pos_embed_file": "",
    "word_embed_matrix_file": "embeddings/word_embed_matrix.txt",
    "char_embed_matrix_file": "embeddings/char_embed_matrix.txt",
    "pos_embed_matrix_file": "embeddings/pos_embed_matrix.txt",
    "embed_dropout": 0.33,
    "lstm_dropout": 0.33,
    "mlp_dropout": 0.33,
    "batch_size": 128,
    "device": "gpu",
    "debug": false
}
```

**Comment by bart2001** [comment]
최고성능
```python
epoch: 8 - loss: 0.15657824277877808 - uas: 0.9620253443717957 - las: 0.9546413421630859 - eval_loss: 0.48692673444747925 - eval_uas: 0.9163461923599243 - eval_las: 0.8920921683311462
# new best UAS!
```
